src/redux/product/types.ts

```diff
@@ -0,0 +1,26 @@
+export type Product = {
+  id: number;
+  title: string;
+  description: string;
+  price: number;
+  image: string;
+  rate: number;
+};
+
+export enum Status {
+  LOADING = 'loading',
+  SUCCESS = 'completed',
+  ERROR = 'error',
+}
+
+export type SearchProductParams = {

```

Comment:
it is the better approach to use Interface instead of type

Reply:
Done.